doc/modules/grid_search.rst

```diff
@@ -139,6 +140,22 @@ For continuous parameters, such as ``C`` above, it is important to specify
 a continuous distribution to take full advantage of the randomization. This way,
 increasing ``n_iter`` will always lead to a finer search.
 
+There is also a log-uniform random variable with
+:class:`utils.fixes.lograndom`, which is continuous version of ``[1, 10, 100,
+1000]`` and are all equally likely with a log-uniform random variable.
+Formally, this means that ``loguniform(10 ** 0, 10 ** 1) == 10**x`` for some
+``x`` that is uniformly distributed between 0 and 1. This is an alias to
```

Comment:
Maybe replace `0`, and `1` with `a` and `b` as currently it's unclear if 0 and 1 have special meaning there.